pkg/sessionctx/variable/sysvar.go

```diff
@@ -3467,10 +3465,6 @@ const (
 	InnodbFastShutdown = "innodb_fast_shutdown"
 	// InnodbLockWaitTimeout is the name for 'innodb_lock_wait_timeout' system variable.
 	InnodbLockWaitTimeout = "innodb_lock_wait_timeout"
-	// SQLLogBin is the name for 'sql_log_bin' system variable.
```

Comment:
Maybe involve PM to decide if we should keep some level of compatibility with MySQL.

For `sql_log_bin`, we can't implement it in TiDB, so we may allow read it and forbid writing to it. For `log_bin`, seems this is a read-only variable in MySQL, we can also let TiDB always return `OFF` for it.

https://dev.mysql.com/doc/refman/8.4/en/replication-options-binary-log.html#sysvar_sql_log_bin

https://dev.mysql.com/doc/refman/8.4/en/replication-options-binary-log.html#sysvar_log_bin

Reply:
TiDB has discarded binlog, which makes it inherently incompatible with MySQL in this aspect. Therefore, keeping the binlog-related parameters is also pointless. It is recommended to remove them as well.